consensus/hotstuff/timeoutcollector/aggregation_test.go

```diff
@@ -0,0 +1,253 @@
+package timeoutcollector
+
+import (
+	"math/rand"
```

Comment:
It would be better to test with different data at each iteration: we could use `math/rand` and call `Seed(time.Now().UnixNano())` OR just use `crypto/rand` instead.